.circleci/config.yml

```diff
@@ -69,6 +102,23 @@ jobs:
               bash gp-localci-client/generate-new-strings-pot.sh $CIRCLE_BRANCH $CIRCLE_SHA1 $CIRCLE_ARTIFACTS/translate
               rm -rf gp-localci-client
             fi
+      - store_artifacts:
+          path: /tmp/artifacts
+
+  #
+  # Test Calypso
+  #
+  test-calypso:
+    parallelism: 6
```

Comment:
According to CircleCI, our plan has 10 containers. Should we decrease `parallelism` to `5` so 2 PRs can run at the same time, or is CircleCI smart enough about that?

Reply:
I'm not sure about our plan or how it works, but there are jobs that are happy to use free containers.

There are plenty of significant improvements we can make in this PR and tweak this number later.

Reply:
We might even play with this then use insights to look at build and queue time over an extended period.